tests/test_filtsmooth/test_gaussfiltsmooth/test_kalman.py

```diff
@@ -124,11 +124,11 @@ def test_rmse_filt_smooth(kalman, data):
     """Assert that smoothing beats filtering beats nothing."""
     obs, times, truth = data
 
-    filter_posterior = kalman.filter(obs, times)
-    smooth_posterior = kalman.smooth(filter_posterior)
+    posterior = kalman.filter(obs, times)
```

Comment:
to be honest, I found the previous test more readable^^ (but no need to change back, that's a _very_ minor thing)

Reply:
weird stuff happening here. It should have been
```python

    posterior = kalman.filtsmooth(obs, times)

    filtms = posterior.filtering_posterior.state_rvs.mean
    smooms = posterior.state_rvs.mean
```
which is what I changed it to now.